roles/edpm_nova/tasks/adoption.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Stop and disable compute services
+  no_log: "{{ edpm_network_config_hide_sensitive_logs | bool }}"
+  tags:
+    - adoption
+    - nova
+  become: true
+  ansible.builtin.shell: |
```

Comment:
can we use ansible.builtin.systemd instead?

Reply:
done

Reply:
have you pushed the updated revision yet?

Reply:
I re-add this shell to simplified checking if service exist from list, task with systemd was failing if service wasn't running